### PR TITLE
Techslate

### PR DESCRIPTION
**Pull request by lavanyanaidu999**

### CODE REVIEW

File: docker-compose/sonar-with-postgres/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+version: '3.7'
+
+services:
+  sonarqube:
+    image: sonarqube:latest
+    ports:
+      - 9000:9000
+    environment:
+      - SONARQUBE_JDBC_USERNAME=sonar
+
```
Comment (techslateramu) [inline docker-compose/sonar-with-postgres/docker-compose.yaml]: no extra space

File: docker-compose/sonar-with-postgres/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      - SONARQUBE_JDBC_USERNAME=sonar
+      
+      - SONARQUBE_JDBC_PASSWORD=sonar
+      - SONARQUBE_JDBC_URL=jdbc:postgresql://db:5432/sonar
+    volumes:
+      - sonarqube_conf:/opt/sonarqube/conf
+      - sonarqube_data:/opt/sonarqube/data
+      - sonarqube_extensions:/opt/sonarqube/extensions
+      - sonarqube_bundled-plugins:/opt/sonarqube/lib/bundled-plugins
+    command: ["./bin/run.sh"]
```
Comment (techslateramu) [inline docker-compose/sonar-with-postgres/docker-compose.yaml]: is this required ?

File: docker-compose/sonar-with-postgres/README.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+```
+version: '3.7'
+services:
+  sonarqube:
+    image: sonarqube:latest
+    ports:
+      - 9000:9000
+    environment:
+      - SONARQUBE_JDBC_USERNAME=sonar
+
```
Comment (techslateramu) [inline docker-compose/sonar-with-postgres/README.md]: remove extra space

File: docker-compose/sonar-with-postgres/README.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+      - SONARQUBE_JDBC_USERNAME=sonar
+      
+      - SONARQUBE_JDBC_PASSWORD=sonar
+      - SONARQUBE_JDBC_URL=jdbc:postgresql://db:5432/sonar
+    volumes:
+      - sonarqube_conf:/opt/sonarqube/conf
+      - sonarqube_data:/opt/sonarqube/data
+      - sonarqube_extensions:/opt/sonarqube/extensions
+      - sonarqube_bundled-plugins:/opt/sonarqube/lib/bundled-plugins
+    command: ["./bin/run.sh"]
```
Comment (techslateramu) [inline docker-compose/sonar-with-postgres/README.md]: remove this line and run again to test, if everything is working. if yes - we dont need this line

File: docker-compose/sonar-with-postgres/README.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+![docker-compose logs](images\logs.png)
+
+**Now open in your browser http://localhost:9000 and login to our default admin account.**
+***
+### **7. Use the default credentials admin:admin to login.**
+
+![Sonar Login](images\Sonarlogin.png)
+
+
+**We have successfully built a SonarQube server.**
```
Comment (techslateramu) [inline docker-compose/sonar-with-postgres/README.md]: write it generic... don't mention we

**Review by techslateramu — CHANGES_REQUESTED** [review]
please fix all the comments and respond when done

File: docker-compose/sonar-with-postgres/README.md (hunk 5 above)
Comment (lavanyanaidu999) [reply docker-compose/sonar-with-postgres/README.md]: Fixed

**Review by lavanyanaidu999 — COMMENTED** [review]
Review Complete

File: docker-compose/sonar-with-postgres/README.md (hunk 4 above)
Comment (lavanyanaidu999) [reply docker-compose/sonar-with-postgres/README.md]: Yes removed it . and runned once again , is running succefully

**Comment by lavanyanaidu999** [comment]
Review Complete

**Comment by techslateramu** [comment]
Approved

**Pull request merged**